### ISSUE TITLE
Settings nullable: false for a string

### ISSUE THREAD

**Issue by jsherman43** [domaindrivendev/Swashbuckle.AspNetCore#1982]
Hi, is it possible to have strings and display nullable: false? It only looks like I can annotate a class with [Required], I just want to display which fields are actually returning back to the user that are guaranteed not to be null ( response schema ).

**Comment by domaindrivendev** [comment]
There's currently no support out-of-the-box for this, but you could easily do it with a simple schema filter (see readme for how to wire these up):

```csharp
    public class UnsetNullableForStringsFilter : ISchemaFilter
    {
        public void Apply(OpenApiSchema schema, SchemaFilterContext context)
        {
            if (schema.Type == "string")
            {
                schema.Nullable = false;
            }
        }
    }
```

**Comment by domaindrivendev** [comment]
UPDATE: It was very simple to add this to the existing `SwaggerSchemaAttribute` so I just went ahead and did that. This will be available as follows in the `6.0.0` release, which I'm planning to get out this week:

```csharp
public class Response
{
    [SwaggerSchema(Nullable = false)]
    public string Foo { get; set; }
}
```
